qemu/tests/cfg/numa_cpu.cfg

```diff
@@ -0,0 +1,82 @@
+- numa_cpu:
+    type = numa_cpu
+    kill_vm = yes
+    no Host_RHEL.m6, Host_RHEL.m7
+    # '-numa cpu' was added from QEMU 3.0 and 'dies' was added from QEMU 4.1
+    # considering libvirt 6.6.0 which consum QEMU 5.1 still have not support
+    # '-numa cpu', so test from QEMU 4.1 is not risky
+    required_qemu = [4.1.0, )
+    mem = 4096
+    ppc64, ppc64le:
+        # ppc use vcpu_dies = INVALID, means 1 when computing cpu topology
+        smp = 8
+        vcpu_sockets = 2
+        vcpu_cores = 2
+        vcpu_threads = 2
+    x86_64, i386:
+        smp = 16
+        vcpu_sockets = 2
+        vcpu_dies = 2
+        vcpu_cores = 2
+        vcpu_threads = 2
+        HostCpuVendor.amd:
+            smp = 8
+            # To adapt all amd system, test signal thread only
+            vcpu_threads = 1
+    vcpu_maxcpus = ${smp}
+    backend_mem = memory-backend-ram
+    use_mem = no
+    variants:
+      - @default:
+          guest_numa_nodes = "node0 node1 node2 node3"
+          mem_devs = "mem0 mem1 mem2 mem3"
+          numa_memdev_node0 = mem-mem0
+          numa_memdev_node1 = mem-mem1
+          numa_memdev_node2 = mem-mem2
+          numa_memdev_node3 = mem-mem3
+          numa_nodeid_node0 = 0
+          numa_nodeid_node1 = 1
+          numa_nodeid_node2 = 2
+          numa_nodeid_node3 = 3
+          size_mem = 1024M
+          guest_numa_cpus = "cpu0 cpu1 cpu2 cpu3"
+          # For ppc, codeid must be a multiple of 'vcpu_threads'
+          ppc64, ppc64le:
+              numa_cpu_nodeid_cpu0 = 0
+              numa_cpu_coreid_cpu0 = 0
+
+              numa_cpu_nodeid_cpu1 = 1
+              numa_cpu_coreid_cpu1 = 2
+
+              numa_cpu_nodeid_cpu2 = 2
+              numa_cpu_coreid_cpu2 = 4
+
+              numa_cpu_nodeid_cpu3 = 3
+              numa_cpu_coreid_cpu3 = 6
+          x86_64, i386:
+              HostCpuVendor.intel:
+                  guest_numa_cpus += " cpu4"
```

Comment:
Move to L77?

Reply:
Updated, thanks!

Reply:
Ah~, yes, just missed, updated now, thanks!